.travis.yml

```diff
@@ -1,8 +1,8 @@
 # All Travis builds are a redundancy against Jenkins failures, except that we also use
 # this for running builds on PRs from forks, because we're OSS friendly. :)
 language: node_js
-matrix:
-  fast_finish: true
+os: linux
+dist: xenial
```

Comment:
Not strictly necessary to put these here explicitly as Travis defaults to them, but think it's better to be explicit.